package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "data-portal",
-  "version": "0.0.0",
+  "version": "1.0.0",
```

Comment:
I would leave it at 0.x until it's released and then directly jump to 2.0 in order to not cause confusion with existing legacy frontend versions when this is tested by the team. But I can change that in my clean-up PR.